### ISSUE TITLE
Add default yml config

### ISSUE THREAD

**Issue by rndquu** [ubiquity/ubiquibot#327]
More info [here](https://github.com/ubiquity/pay.ubq.fi/issues/68)

What should be done:
1. Add a default bot's `yml`config so that the [config generation page](https://pay.ubq.fi/) could fetch it via github API
2. Assign the default values [here](https://github.com/ubiquity/ubiquibot/blob/development/src/bindings/config.ts#L15) from the default config `yml` file instead of the hardcoded param values

**Comment by Venoox** [comment]
I'm a bit confused. For example [here](https://github.com/ubiquity/ubiquibot/blob/15464d62a88d93f971b9a3884a22e633d121305c/src/bindings/config.ts#L15) the timeLabels is on second level in price object but in [here]([https://github.com/ubiquity/pay.ubq.fi/blob/9efa53d56f8a7f56ac93ce12b701d28bfcc4a5ea/static/scripts/secret.ts#L48) timeLabels is on first level, in general the ubiquibot config has a lot of nesting but pay.ubq.fi config is only 1 level

**Comment by rndquu** [comment]
> I'm a bit confused. For example [here](https://github.com/ubiquity/ubiquibot/blob/15464d62a88d93f971b9a3884a22e633d121305c/src/bindings/config.ts#L15) the timeLabels is on second level in price object but in [here](https://github.com/ubiquity/pay.ubq.fi/blob/9efa53d56f8a7f56ac93ce12b701d28bfcc4a5ea/static/scripts/secret.ts#L48) timeLabels is on first level, in general the ubiquibot config has a lot of nesting but pay.ubq.fi config is only 1 level

This [one](https://github.com/ubiquity/ubiquibot/blob/15464d62a88d93f971b9a3884a22e633d121305c/src/bindings/config.ts#L15) is internal js config representation (multilevel)

This [one](https://github.com/ubiquity/pay.ubq.fi/blob/9efa53d56f8a7f56ac93ce12b701d28bfcc4a5ea/static/scripts/secret.ts#L48) is an interface used for [yml config](https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml) (single level)

So the original yml config is single level while in js it is represented as a "multilevel" object (plain js object)

So basically you should: 
1. create a default yml config (like this [one](https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml)) in the bot's [repo](https://github.com/ubiquity/ubiquibot)
2. read default values from the default yml config [here](https://github.com/ubiquity/ubiquibot/blob/15464d62a88d93f971b9a3884a22e633d121305c/src/bindings/config.ts#L15)

**Comment by 0xcodercrane** [comment]
found an issue on the PR. https://github.com/ubiquity/ubiquibot/pull/366#issuecomment-1577753888

**Comment by pavlovcik** [comment]
What's the update here?

**Comment by rndquu** [comment]
@0xcodercrane shouldn't we assign Venoox and open/close the issue again for the permit to be generated?

**Comment by 0xcodercrane** [comment]
> @0xcodercrane shouldn't we assign Venoox and open/close the issue again for the permit to be generated?

I think we should! let me handle it.

**Comment by 0xcodercrane** [comment]
@Venoox ^^ here is your claim url.
